examples/user_guide/Plotting.ipynb

```diff
@@ -173,19 +173,19 @@
     "\n",
     "Plot types available include:\n",
     "\n",
-    "* <a href=\"#Area\">``.area()``</a>: Plots a  area chart similar to a line chart except for filling the area under the curve and optionally stacking \n",
-    "* <a href=\"#Bars\">``.bar()``</a>: Plots a bar chart that can be stacked or grouped\n",
-    "* <a href=\"#Bivariate\">``.bivariate()``</a>: Plots 2D density of a set of points \n",
-    "* <a href=\"#Box-Whisker-Plots\">``.box()``</a>: Plots a box-whisker chart comparing the distribution of one or more variables\n",
-    "* <a href=\"#HeatMap\">``.heatmap()``</a>: Plots a heatmap to visualizing a variable across two independent dimensions\n",
-    "* <a href=\"#HexBins\">``.hexbins()``</a>: Plots hex bins\n",
-    "* <a href=\"#Histogram\">``.hist()``</a>: Plots the distribution of one or histograms as a set of bins\n",
-    "* <a href=\"#KDE\">``.kde()``</a>: Plots the kernel density estimate of one or more variables.\n",
-    "* <a href=\"#The-plot-method\">``.line()``</a>: Plots a line chart (such as for a time series)\n",
-    "* <a href=\"#Scatter\">``.scatter()``</a>: Plots a scatter chart comparing two variables\n",
-    "* <a href=\"#Step\">``.step()``</a>: Plots a step chart akin to a line plot\n",
-    "* <a href=\"#Tables\">``.table()``</a>: Generates a SlickGrid DataTable\n",
-    "* <a href=\"#Violin-Plots\">``.violin()``</a>: Plots a violin plot comparing the distribution of one or more variables using the kernel density estimate"
+    "* <a href=\"#area\">``.area()``</a>: Plots a  area chart similar to a line chart except for filling the area under the curve and optionally stacking \n",
+    "* <a href=\"#bars\">``.bar()``</a>: Plots a bar chart that can be stacked or grouped\n",
+    "* <a href=\"#bivariate\">``.bivariate()``</a>: Plots 2D density of a set of points \n",
+    "* <a href=\"#box-whisker-plots\">``.box()``</a>: Plots a box-whisker chart comparing the distribution of one or more variables\n",
+    "* <a href=\"#heatmap\">``.heatmap()``</a>: Plots a heatmap to visualizing a variable across two independent dimensions\n",
+    "* <a href=\"#hexbins\">``.hexbins()``</a>: Plots hex bins\n",
+    "* <a href=\"#histogram\">``.hist()``</a>: Plots the distribution of one or histograms as a set of bins\n",
+    "* <a href=\"#kde-density\">``.kde()``</a>: Plots the kernel density estimate of one or more variables.\n",
+    "* <a href=\"#the-plot-method\">``.line()``</a>: Plots a line chart (such as for a time series)\n",
+    "* <a href=\"#the-plot-method\">``.scatter()``</a>: Plots a scatter chart comparing two variables\n",
```

Comment:
```suggestion
    "* <a href=\"#scatter\">``.scatter()``</a>: Plots a scatter chart comparing two variables\n",
```

(I guess this is up for discussion given that it's discussed in both places, but linking to the section only about Scatter seems less confusing, I think.)